src/pybind/mgr/volumes/volume.py

```diff
@@ -156,16 +263,43 @@ def volume_list(self):
             result.append({'name': f['mdsmap']['fs_name']})
         return 0, json.dumps(result, indent=2), ""
 
+    def connection_pool_wrap(func):
```

Comment:
The decorator should probably have been documented -- I'll have that up with next push...